src/test/java/bio/terra/service/dataset/flight/create/DatasetCreateFlightTest.java

```diff
@@ -40,16 +42,26 @@ void testDatasetCreateFlightAuthorizeBillingSteps(boolean isTDRBillingProfile) {
     inputParameters.put(JobMapKeys.TDR_BILLING_PROFILE_FALLBACK.getKeyName(), isTDRBillingProfile);
     var flight = new DatasetCreateFlight(inputParameters, context);
 
+    List<String> stepNames = FlightTestUtils.getStepNames(flight);
+
     if (isTDRBillingProfile) {
       assertThat(
           "Dataset creation flight locates the billing info and then has two optional steps",
-          FlightTestUtils.getStepNames(flight),
+          stepNames,
           containsInRelativeOrder("AuthorizeBillingProfileUseStep"));
```

Comment:
`hasItem()` is a more direct way to test for a single item in a collection
```suggestion
          hasItem("AuthorizeBillingProfileUseStep"));
```